reference-implementation/run-web-platform-tests.js

```diff
@@ -19,6 +19,9 @@ wptRunner(testsPath, setup)
   });
 
 function setup(window) {
+  // Necessary so that we can send test-realm promises to the jsdom-realm implementation without causing assimilation.
+  window.Promise = Promise;
```

Comment:
Mehh. wpt-runner is kind of a giant hack, so it might not be worth your time.

It basically creates a whole nother global environment in which to run its scripts, including new values for Promise, Map, Set, Number, etc. This usually isn't a problem, but for promises it causes problems now that V8 has updated their promise-detection code.